includes/class-wcs-cart-initial-payment.php

```diff
@@ -52,6 +52,23 @@ public function maybe_setup_cart() {
 			return;
 		}
 
+		/**
+		 * Filter whether to recreate the initial payment order.
```

Comment:
* Add `@since` tag once known.
* Is `wcs_recreate_initial_payment_order` the best name?